src/core/reopt_inputs.jl

```diff
@@ -650,8 +650,16 @@ function setup_absorption_chiller_inputs(s::AbstractScenario, max_sizes, min_siz
         cap_cost_slope["AbsorptionChiller"] = s.absorption_chiller.installed_cost_per_kw
     end
 
-    cop["AbsorptionChiller"] = s.absorption_chiller.chiller_elec_cop
-    thermal_cop["AbsorptionChiller"] = s.absorption_chiller.chiller_cop
+    cop["AbsorptionChiller"] = s.absorption_chiller.cop_electric
+    if isnothing(s.chp)
+        thermal_factor = 1.0
+    elseif s.chp.cooling_thermal_factor == 0.0
+        error("The CHP cooling_thermal_factor is 0.0 which implies that CHP cannot serve AbsorptionChiller. If you
+            want to model CHP and AbsorptionChiller, you must specify a cooling_thermal_factor greater than 0.0")
```

Comment:
Do we want to support a case with a CHP system that can meet heating load but not serve the absorption chiller (which could be served by the boiler), in which case we'd switch `error` with `warn`?

Reply:
I agree that should/could be a possibility, but that would be strange to put in a CHP system and not have it be able to serve the absorption chiller.